bin/i18n/sync-all.rb

```diff
@@ -152,8 +152,10 @@ def return_to_staging_branch
       # so error out.
       raise "Tried to return to staging branch from unknown branch #{GitUtils.current_branch.inspect}"
     end
-
     `git checkout staging` if should_i "switch to staging branch"
+  rescue => e
+    puts "return_to_staging_branch failed from the error: #{e}"
+    raise e
```

Comment:
Does this exception reported in Honeybadger?

Reply:
No it doesn't. I think I would prefer that it would ping @foundations-team in slack when the build failed, but I don't know how slack integration works at the moment. I just wanted to get something quick out. If you know how to, please setup a call with me!